### PR TITLE
Reformat code

### LINKED ISSUE #338: Reformat Code
**Problem**
The code contains a lot of inconsistencies with tabs being used in places and spaces in others, various line endings, indentations and other style formats. We should standardize this so we're all on the same format and not causing unnecessary churn on the code due to whitespace changes.

**Solution**
We now have [EditorConfig](https://editorconfig.org) support checked in and enforces some basic formats for all IDEs and some specific formats for just Intellij. We should use Intellij's ability to reformat all code at once and create a PR that contains only format changes. It won't fix everything and it won't enforce all standards across all IDEs, but it's a start.

**Alternatives**
- Leave as is
- Fix as we go

**Additional Context**

### PR DESCRIPTION
**Pull request by steven-sheehy**
**Detailed description**:
- Uses Intellij's code reformat option to reformat all Java code per the settings in the .editorconfig.
- Since Intellij community version doesn't have Javascript support, used prettier to manually reformat code
- Integrated prettier as a git commit hook that runs automatically on commit and cleans up any changed files

**Which issue(s) this PR fixes**:
Fixes #338 

**Special notes for your reviewer**:
Ignore whitespace when looking at diff

**Checklist**
- [ ] Documentation added
- [x] Tests updated

### CODE REVIEW

File: rest-api/__acceptancetests__/acceptancetest_utils.js [hunk 1]
```diff
@@ -4,48 +4,51 @@ const config = require('../config.js');
...
  * Converts seconds since epoch (seconds.nnnnnnnnn format) to  nanoseconds
- * @param {String} Seconds since epoch (seconds.nnnnnnnnn format) 
+ * @param {String} Seconds since epoch (seconds.nnnnnnnnn format)
  * @return {String} ns Nanoseconds since epoch
  */
-const secNsToNs = function (secNs) {
-    return (math.multiply(math.bignumber(secNs), math.bignumber(1e9)).toString());
-}
+const secNsToNs = function(secNs) {
+  return math.multiply(math.bignumber(secNs), math.bignumber(1e9)).toString();
```
Comment (Nana-EC) [inline rest-api/__acceptancetests__/acceptancetest_utils.js]: q: does the wrapping go after a max number of chars?
I'm curious why nsToSecNs() got every method call wrapped to a line and this method secNsToNs() didn't.
I guess I'm more curious if we are trying to establish a standard of new line per nested method call or not.

File: rest-api/monitoring/monitor_dashboard/js/main.js [hunk 2]
```diff
@@ -51,34 +51,46 @@ const init = () => {
...
-}
+  data.results.testResults.forEach(result => {
+    // Skip the 'Skipped tests' that are marked as pending in jest json output
+    if (result.result === 'pending') {
+      return;
+    }
+    const failureMsg =
+      result.failureMessages == undefined ? '' : result.failureMessages.join('<br>').replace('\n', '<br>');
+
+    h +=
```
Comment (Nana-EC) [inline rest-api/monitoring/monitor_dashboard/js/main.js]: I actually don't like this. HTML like other languages is indented by definition and reading it is clearer with that indentation. If we're formatting it should to align tags not make every tag align to the beginning.

File: rest-api/__acceptancetests__/acceptancetest_utils.js (hunk 1 above)
Comment (steven-sheehy) [reply rest-api/__acceptancetests__/acceptancetest_utils.js]: It should be 120 characters as specified in the .editorconfig

File: rest-api/monitoring/monitor_dashboard/js/main.js (hunk 2 above)
Comment (steven-sheehy) [reply rest-api/monitoring/monitor_dashboard/js/main.js]: I don't think is something we can control as this is a very specialized use of HTML embedded in javascript and not using templating. This should probably be converted to a single template string.

**Pull request merged**